iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/bucket/OrganizationSectionHasher.java

```diff
@@ -0,0 +1,91 @@
+package eu.dnetlib.iis.wf.affmatching.bucket;
+
+import java.io.Serializable;
+
+import org.apache.commons.lang3.StringUtils;
+
+import eu.dnetlib.iis.wf.affmatching.orgsection.OrganizationSection;
+
+/**
+ * Class that generates hash of the given {@link OrganizationSection}.
+ * 
+ * @author madryk
+ */
+public class OrganizationSectionHasher implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private static final int SECTION_NUMBER_OF_LETTERS = 3;
```

Comment:
I suppose this number depends on the number of letters required to disambiguate different section types? Then maybe it should be a constant in `OrganizationSection`?